### PR TITLE
Fix exosphere-shared app templates

### PR DESCRIPTION
**Pull request by trushton**
**partially** resolves #19

These changes to the app-templates connect the templates to mongo running in docker correctly.
With these merged in, the `tutorial.feature` of the `cli` will be will pass and be ready for publishing.

@kevgo @hugobho @martinjaime

### CODE REVIEW

File: exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = function() {
 
   this.Given(/^an instance of this service$/, function(done) {
     this.process = new ExoService({ serviceName: '_____serviceName_____',
-                                    exocomHost: 'localhost'
+                                    exocomHost: 'localhost',
```
Comment (kevgo) [inline exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js:27]: commas are optional in LiveScript, you can remove this one and the one on the line above

File: exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/support/env.js [hunk 2]
```diff
@@ -29,7 +29,6 @@ module.exports = function() {
   this.After(function() {
     this.exocom && this.exocom.close()
     this.process && this.process.close()
-    this.process && this.process.closePort()
```
Comment (kevgo) [inline exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/support/env.js:32]: This change is not related to WebPack. Extract into another PR with explanation what it achieves?

File: exosphere-shared/templates/add-service/exoservice-es6-mongodb/src/server.js [hunk 3]
```diff
@@ -10,7 +10,11 @@ module.exports = {
 
   beforeAll: (done) => {
     const mongoDbName = `exosphere-_____serviceName_____-${env}`
-    MongoClient.connect(`mongodb://localhost:27017/${mongoDbName}`, N( (mongoDb) => {
+    var mongoAddress = 'mongo'
```
Comment (kevgo) [inline exosphere-shared/templates/add-service/exoservice-es6-mongodb/src/server.js]:
```js
const mongoAddress = (env === 'test') ? 'localhost' : 'mongo'

**Review by kevgo — APPROVED** [review]

File: exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js (hunk 1 above)
Comment (trushton) [reply exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js:27]: This is the es6 template, they are required here aren't they?

Comment (kevgo) [reply exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/steps/steps.js:27]: Yes, hahaha!
